dan_layer/template_test_tooling/src/template_test.rs

```diff
@@ -474,12 +474,14 @@ impl TemplateTest {
             initial_ownership_proofs: proofs,
         };
         let processor = TransactionProcessor::new(
+            TransactionProcessorConfigBuilder::new()
```

Comment:
nit: A nice pattern is to return the builder from the type being built

```rust
TransactionProcessorConfig::builder().with_network(..).build()
```

Reply:
yepp true, updated